pynest/examples/wang_neuron.py

```diff
@@ -0,0 +1,101 @@
+import nest
```

Comment:
This example lacks comments and the copyright header. The example should explain how to record spikes (see comment in test below).